editor/svgcanvas.js

```diff
@@ -186,9 +186,12 @@ var selectedElements = [];
 // * element - tag name of the SVG element to create
 // * attr - Object with attributes key-values to assign to the new element
 // * curStyles - Boolean indicating that current style attributes should be applied first
+// * children - Optional array with data objects to be added recursively as children
 //
 // Returns: The new element
 var addSvgElementFromJson = this.addSvgElementFromJson = function(data) {
+	if(typeof(data) == 'string') return svgdoc.createTextNode(data);
```

Comment:
I had difficulty finding a coding style to adhere to. There were instances of both with and without the space, as well as things like indentation in some places with tabs and in some places with spaces. I've added a commit fixing this and i'll try to follow this style in future contributions.